changelog.yaml

```diff
@@ -51,6 +51,14 @@
 - version: NEXT
   date: TBD
   changes:
+    - type: internal
+      impact: patch
+      title: Increase default log level
+      description: |-
+        Default log level was increaset to 3.
+        Small adjustments to log output.
+      pullRequestNumber: 0
+      jiraIssueNumber: CLOUDCIFEAT1-173
```

Comment:
Interna... but only the number would not work since the project is not the default one 🙄